README.md

```diff
@@ -19,6 +19,18 @@ the UI framework used in [Fuse](https://www.fusetools.com/) apps.
 * `build.sh` runs stuff (which downloads uno if needed), and builds all packages.
 * `test.sh` runs all tests.
 
+
+### Fuse
+
+If you want to use a self-built fuselibs installation with an installed Fuse, you can do this through a `.unoconfig`-file, something like this:
```

Comment:
@kusma Oh sorry, didn't see that – skipped right to the diff on this one.

I think we should document both options, as they can both be useful in different cases. We can always add a sentence about running uno from the repo directory if we decide to reinstate the repo-level .unoconfig.

How about something like this?

---

You may use a locally built copy of fuselibs with an installed Fuse. This is done by creating a file named `.unoconfig` in either a Fuse project directory (applies to that project only), or in your home directory (applies to all projects). It should contain something like the following: